src/app/settings/stix-objects/attack-patterns/attack-pattern-list/attack-pattern-list.component.spec.ts

```diff
@@ -1,3 +1,34 @@
+<<<<<<< HEAD
```

Comment:
Can you resolve these commit merge markers

Reply:
This should be fix